README.md

```diff
@@ -224,6 +225,38 @@ found in 'open-source-firmware-validation' folder.
 * [Renaming Test Cases](https://robotidy.readthedocs.io/en/stable/transformers/RenameTestCases.html)
 * [Renaming Variables](https://robotidy.readthedocs.io/en/stable/transformers/RenameVariables.html)
 
+## Generating documentation
```

Comment:
@WiktorG351, let's update this section after the successful integration of the gh pages

Reply:
@artur-rs added [here](https://github.com/Dasharo/open-source-firmware-validation/pull/352/commits/0e9c71469d6f983f1c8c0bd884634526ba577e2d)